packages/storybook/stories/components/tabs/migration.from.17.x.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta } from '@storybook/blocks';
```

Comment:
Kepp the old file (just update Badge), this is the 17 to 18 migration, not the React one